src/queryBuilder/QueryBuilder.js

```diff
@@ -36,28 +35,32 @@ export default DDContext(
       mapNodeToProps,
       MissingTypeComponent,
     }) => (
-      <Provider
-        ContextureButton={Button}
-        {...{ fields, types, mapNodeToProps, MissingTypeComponent }}
-      >
-        <div style={{ background }}>
-          {state.getNode(path) && (
-            <Group
-              node={state.getNode(path)}
-              tree={state}
-              isRoot={true}
-              {...{ Button, Modal, Picker }}
-            />
-          )}
-          <Button
-            onClick={() => {
-              state.adding = !state.adding
+      <div style={{ background }}>
+        {state.getNode(path) && (
+          <Group
+            node={state.getNode(path)}
+            tree={state}
+            isRoot={true}
+            ContextureButton={Button}
+            {...{
+              Button,
+              Modal,
+              Picker,
+              fields,
+              types,
+              mapNodeToProps,
```

Comment:
These are not directly used by `Group` nor are you passing them along. This likely breaks all of these cases in advanced search. To be safe, we should just switch to a native React context - prop drilling isn't really better here.